core-web/libs/sdk/client/src/lib/client/sdk-js-client.spec.ts

```diff
@@ -77,6 +77,94 @@ describe('DotCmsClient', () => {
+                mockFetchResponse(mockResponse);
+
+                const client = dotcmsClient.init(config);
+
+                await client.page.get({ path: '/home' });
+
+                expect(fetch).toHaveBeenCalledWith(
+                    'https://example.com/api/v1/page/json/home?host_id=123456',
+                    {
+                        headers: { Authorization: 'Bearer ABC' }
+                    }
+                );
+            });
+
+            it('should use a clean dotcmsUrl when passing a route at the end', async () => {
+                const config = {
+                    dotcmsUrl: 'https://example.com/some/cool/route',
+                    siteId: '123456',
+                    authToken: 'ABC'
+                };
+
+                const mockResponse = { content: 'Page data' };
+                mockFetchResponse(mockResponse);
+
+                const client = dotcmsClient.init(config);
+
+                await client.page.get({ path: '/home' });
+
+                expect(fetch).toHaveBeenCalledWith(
+                    'https://example.com/api/v1/page/json/home?host_id=123456',
+                    {
+                        headers: { Authorization: 'Bearer ABC' }
+                    }
+                );
+            });
+
+            it('should use a clean dotcmsUrl when passing a port and an slash at the end', async () => {
+                const config = {
+                    dotcmsUrl: 'https://example.com:3434/cool/route',
+                    siteId: '123456',
+                    authToken: 'ABC'
+                };
+
+                const mockResponse = { content: 'Page data' };
+                mockFetchResponse(mockResponse);
+
+                const client = dotcmsClient.init(config);
+
+                await client.page.get({ path: '/home' });
+
+                expect(fetch).toHaveBeenCalledWith(
+                    'https://example.com:3434/api/v1/page/json/home?host_id=123456',
+                    {
+                        headers: { Authorization: 'Bearer ABC' }
+                    }
+                );
+            });
+
+            it('should use a clean dotcmsUrl when passing a port and an slash at the end', async () => {
+                const config = {
+                    dotcmsUrl: 'https://example.com:3434/',
+                    siteId: '123456',
+                    authToken: 'ABC'
+                };
+
+                const mockResponse = { content: 'Page data' };
+                mockFetchResponse(mockResponse);
+
+                const client = dotcmsClient.init(config);
+
+                await client.page.get({ path: '/home' });
+
+                expect(fetch).toHaveBeenCalledWith(
+                    'https://example.com:3434/api/v1/page/json/home?host_id=123456',
+                    {
+                        headers: { Authorization: 'Bearer ABC' }
+                    }
+                );
+            });
```

Comment:
@zJaaal Missing unit test to verify  `toThrow("Invalid configuration - 'dotcmsUrl' must be a valid URL");`

Reply:
It is there actually in [Line 57](https://github.com/dotCMS/core/blob/28932-uve-config-for-dotcms_host-should-support-at-the-end-of-the-url/core-web/libs/sdk/client/src/lib/client/sdk-js-client.spec.ts#L57)